CONTRIBUTING.md

```diff
@@ -66,9 +66,11 @@ Useful git commands that can help a lot with this are `git commit --patch --amen
 From time to time, changes between branches must be rebased, for example, if the
 number of new rebuilds they would cause is too large for the target branch. When
 rebasing, care must be taken to include only the intended changes, otherwise
-many CODEOWNERS will be inadvertently requested for review.  To achieve this,
+many CODEOWNERS will be inadvertently requested for review. To achieve this,
 rebasing should not be performed directly on the target branch, but on the merge
-base between the current and target branch.
+base between the current and target branch. As an additional precautionary measure,
+you should temporarily mark the PR as draft for the duration of the operation.
+Draft PRs won't ping any people.
```

Comment:
They won't ping anyone through codeowners, but ofborg will still request people (iirc).

Reply:
Yeah that came up recently, IMO that's an ofborg problem though. I should mention it in the text though.